concept/type/owns.feature

```diff
@@ -0,0 +1,3952 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#noinspection CucumberUndefinedStep
+Feature: Concept Owns
+
+  # TODO: Refactor "DEPRECATED"!
+  Background:
+    Given typedb starts
+    Given connection opens with default authentication
+    Given connection has been opened
+    Given connection does not have any database
+    Given connection create database: typedb
+    Given connection open schema transaction for database: typedb
+
+    Given put entity type: person
```

Comment:
I create `entity` and `relation` types in `Background` so I could use it inside `Scenario Outlines` to test both `entity` and `relation` owning an `attribute` within one `Scenario`!